### PR TITLE
gpu: Prevent false positives for aliased bindings

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Prevents false positives - tracking issue https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/8875

### CODE REVIEW

File: tests/unit/gpu_av.cpp [hunk 1]
```diff
@@ -1301,3 +1301,72 @@ TEST_F(NegativeGpuAV, CopyBufferToImageD32U8Vk13) {
...
+        #extension GL_EXT_samplerless_texture_functions : require
+
+        layout(set = 0, binding = 0) uniform texture2DMS BaseTextureMS;
+        layout(set = 0, binding = 0) uniform texture2D BaseTexture;
+        layout(set = 0, binding = 1) uniform sampler BaseTextureSampler;
+        layout(set = 0, binding = 2) buffer SSBO { vec4 dummy; };
+        layout (constant_id = 0) const int path = 0; // always zero, but prevents dead code elimination
+
+        void main() {
+            dummy = texture(sampler2D(BaseTexture, BaseTextureSampler), vec2(0));
```
Comment (arno-lunarg) [inline tests/unit/gpu_av.cpp:1324]: Not sure why this `dummy` assignment is not considered dead code, and the following `+=` is, I trust your judgment on this!

**Review by arno-lunarg — APPROVED** [review]

File: layers/drawdispatch/descriptor_validator.cpp [hunk 2]
```diff
@@ -340,6 +340,14 @@ bool DescriptorValidator::ValidateDescriptor(const DescriptorBindingInfo &bindin
         return skip;
     }
 
+    // https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/8875
+    // TODO - The logic to find the OpVariable is broken when two OpVariable are aliased, for now to prevent false positives here
+    // until we can get more info out of GPU-AV to know which OpVariable was actually used. (Skipping here because the alias doesn't
+    // effect the above checks if the descriptor is valid to access or not)
+    if (binding_info.second.size() > 1) {
```
[the remark this answers is not in the document]
Comment (spencer-lunarg) [reply layers/drawdispatch/descriptor_validator.cpp:347]: yes, it does, I already have ideas, but wanted to keep this PR simple

File: tests/unit/gpu_av.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply tests/unit/gpu_av.cpp:1324]: both are not dead code

**Comment by spencer-lunarg** [comment]
> CI Vulkan-ValidationLayers build # 18047 failed.

False positives from Mac, all tests passed

**Pull request merged**